src/app/SharpRaven/Data/SentryRequestFactory.cs

```diff
@@ -35,7 +35,9 @@
 using System.Linq;
 using System.Reflection;
 using System.Text;
-
+#if net35
+using System.Web;
```

Comment:
Could the reference to `System.Web` be avoided if the `HttpContext` was treated as an `object` and then have its properties accessed through reflection instead? Too much work?